WooCommerce/Classes/ViewRelated/Orders/Order Creation/CollapsibleProductRowCard.swift

```diff
@@ -47,6 +52,49 @@ struct CollapsibleProductRowCard: View {
                 Text(Localization.priceLabel)
                 CollapsibleProductCardPriceSummary(viewModel: viewModel)
             }
+            .padding(.bottom)
+            HStack {
+                if viewModel.discount == nil {
+                    Button(Localization.addDiscountLabel) {
+                        onAddDiscount()
+                    }
+                    .buttonStyle(PlusButtonStyle())
+                } else {
+                    HStack {
+                        Button(action: {
+                            onAddDiscount()
+                        }, label: {
+                            HStack {
+                                Text(Localization.discountLabel)
+                                Image(uiImage: .pencilImage)
+                                    .resizable()
+                                    .frame(width: Layout.iconSize, height: Layout.iconSize)
```

Comment:
Idle thought – perhaps we should have a simple view for our GridIcons that neatens this up? Doesn't really need to be in this PR but this boilerplate is a bit of a paper cut compared to using SFSymbols (which I still think are better!)

Reply:
That sounds a good idea, we seem to redo this view a lot and it's indeed some boilerplate that could be reduced. I'll log this on https://github.com/woocommerce/woocommerce-ios/issues/10863 to keep track of it. I can take care of it as part of the project or separately.